common/common_functions.sh

```diff
@@ -126,6 +126,10 @@ get_distro_version(){
     grep ^VERSION_ID= /etc/os-release | cut -d'=' -f2 -s | tr -d '"'
 }
 
+get_kind_distro_version(){
+    echo "20.10"
```

Comment:
Please, add a note that 20.10 is required for kind only

Reply:
will do

Reply:
done, please have a look